[FW][FIX] sipreco_subsidy_management: expedient wizard

fw-bot-adhoc opened the pull request:
Forward-Port-Of: ingadhoc/odoo-public-administration#539

mav-adhoc committed:
[FIX] sipreco_subsidy_management: expedient wizard

X-original-commit: cbaab49535f18431418151cf79a34c897f87dcd0

roboadhoc commented:
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/odoo-public-administration/pull/540).

fw-bot-adhoc commented:
@mav-adhoc @jcadhoc cherrypicking of pull request ingadhoc/odoo-public-administration#539 failed.

stdout:
```
CONFLICT (modify/delete): sipreco_subsidy_management/wizards/create_administrative_process_wizard_views.xml deleted in HEAD and modified in 9de3eee ([FIX] sipreco_subsidy_management: expedient wizard). Version 9de3eee ([FIX] sipreco_subsidy_management: expedient wizard) of sipreco_subsidy_management/wizards/create_administrative_process_wizard_views.xml left in tree.
CONFLICT (modify/delete): sipreco_subsidy_management/wizards/create_administrative_process_wizard.py deleted in HEAD and modified in 9de3eee ([FIX] sipreco_subsidy_management: expedient wizard). Version 9de3eee ([FIX] sipreco_subsidy_management: expedient wizard) of sipreco_subsidy_management/wizards/create_administrative_process_wizard.py left in tree.
Auto-merging sipreco_subsidy_management/__manifest__.py
CONFLICT (content): Merge conflict in sipreco_subsidy_management/__manifest__.py

```

stderr:
```
19:21:28.282740 git.c:444               trace: built-in: git cherry-pick 9de3eee84e593b2420303aefa13d6944a9a6bbf0
error: could not apply 9de3eee... [FIX] sipreco_subsidy_management: expedient wizard
hint: after resolving the conflicts, mark the corrected paths
hint: with 'git add <paths>' or 'git rm <paths>'
hint: and commit the result with 'git commit'
----------
status:

```

Either perform the forward-port manually (and push to this branch, proceeding as usual) or close this PR (maybe?).

In the former case, you may want to edit this PR message as well.

More info at https://github.com/odoo/odoo/wiki/Mergebot#forward-port
